src/liballoc/string.rs

```diff
@@ -1993,6 +2111,28 @@ impl AddAssign<&str> for String {
     }
 }
 
+/// Implements the `+=` operator for appending to a `String`.
+///
+/// This has the same behavior as the [`push_str`][String::push_str] method.
+#[stable(feature = "stringaddassign_string", since = "1.41.0")]
```

Comment:
I guess it would be fine to use the feature name `extra_string_add_impls` or something like that everywhere. If we wanna keep the names you chose, I'd think that they should be proper snake case: `string_add_assign_string`.

Reply:
Yes, please check the new commit.